nixos/modules/services/networking/avahi-daemon.nix

```diff
@@ -39,215 +37,241 @@ let
     enable-reflector=${yesNo reflector}
     ${extraConfig}
   '';
-
 in
-
 {
+  options.services.avahi = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Whether to run the Avahi daemon, which allows Avahi clients
+        to use Avahi's service discovery facilities and also allows
+        the local machine to advertise its presence and services
+        (through the mDNS responder implemented by `avahi-daemon').
+      '';
+    };
 
-  ###### interface
+    hostName = mkOption {
+      type = types.str;
+      default = config.networking.hostName;
+      defaultText = "config.networking.hostName";
```

Comment:
You can use `literalExample "config.networking.hostName"` in order for it not to render as a string, but the literal expression in the manual.

Reply:
@Infinisil isn't `literalExample` just for examples? I'm setting the defaultText for the option here because I want to avoid having the manual rebuilt for changing hostnames. I'm not sure how you meant that :)

Reply:
Yeah I also thought `literalExamples` was just for examples, and clearly that would be intended from its name. But upon checking, it seems to work for `defaultText` as well, and makes the manual look more correct with this. Feel free to add or not